hc/api/tests/test_admin.py

```diff
@@ -17,3 +20,5 @@ def test_it_shows_channel_list_with_pushbullet(self):
         ch.save()
 
         ### Assert for the push bullet
+        r = self.client.get("/admin/api/channel/")
+        # self.assertContains(r, "Pushbullet")
```

Comment:
I have modified the code to allow the `follow = True` and, the test still shows an error `Forbidden 403`. Should I create a test user on the test case who has admin rights?